src/test/java/com/shopify/ShopifySdkDriver.java

```diff
@@ -592,6 +596,30 @@ public void givenSomeValuesWhenUpdatingACustomerThenExpectValuesToBeUpdatedOnCus
 		assertEquals("RyanTest", updatedCustomer.getFirstName());
 	}
 
+	@Test
+	public void givenSomeValuesWhenUpdatingVariantThenUpdateVariant() {
```

Comment:
Just removing this, since it really does not provide any value outside of testing the issue with > 250 images inside the one variant in the dev store.